STEP/CodeGeneration/CodeGenerationVisitor.cs

```diff
@@ -605,8 +603,8 @@ public void Visit(ProgNode n)
         // TODO: library inclusion?
         n.VarsBlock?.Accept(this);
         n.FuncsBlock?.Accept(this);
-        n.SetupBlock?.Accept(this);
-        n.LoopBlock?.Accept(this);
+        n.SetupBlock.Accept(this);
+        n.LoopBlock.Accept(this);
```

Comment:
Skal de ikke stadig tjekkes for null, som med Vars- og FuncBlock?

Reply:
jov 🤦‍♂️